readme.txt

```diff
@@ -1,5 +1,5 @@
 ﻿=== Medium ===
-Contributors: mediumdotcom, majelbstoat, huckphin, amyquispe
+Contributors: mediumdotcom, majelbstoat, huckphin, amyquispe, tangrufus
```

Comment:
Please remove this, as I believe it will give you access to publish and edit new public versions of the plugin.